Cargo.toml

```diff
@@ -149,6 +149,7 @@ tracing-core = { git = "https://github.com/element-hq/tracing.git", rev = "ca943
 tracing-subscriber = { git = "https://github.com/element-hq/tracing.git", rev = "ca9431f74d37c9d3b5e6a9f35b2c706711dab7dd" }
 tracing-appender = { git = "https://github.com/element-hq/tracing.git", rev = "ca9431f74d37c9d3b5e6a9f35b2c706711dab7dd" }
 paranoid-android = { git = "https://github.com/element-hq/paranoid-android.git", rev = "69388ac5b4afeed7be4401c70ce17f6d9a2cf19b" }
+vodozemac = { git = "https://github.com/uhoreg/vodozemac.git", branch = "dehydration_format" }
```

Comment:
@poljar do we need to make a new release of vodozemac, or should I just set the vodozemac dependency to a specific git rev?

Reply:
You can pin to the git rev, I'll make a release when we need it for the SDK release.

Reply:
I've pinned to the git rev.  What do I need to do to make https://github.com/matrix-org/matrix-rust-sdk/actions/runs/12876768825/job/35900251251?pr=4421 happy?